pymanifold/lib.py

```diff
@@ -119,35 +229,118 @@ def _create_market(
         closeTime: int,
         tags: Optional[List[str]] = None,
         initialProb: Optional[int] = 50,
+        initialValue: Optional[float] = None,
         minValue: Optional[int] = None,
         maxValue: Optional[int] = None,
-    ):
-        """
-        Create a market
-        """
-
+        isLogScale: Optional[bool] = None,
+        answers: Optional[Sequence[str]] = None,
+    ) -> LiteMarket:
+        """Create a market."""
         data = {
             "outcomeType": outcomeType,
             "question": question,
             "description": description,
             "closeTime": closeTime,
-            "tags": tags,
+            "tags": tags or [],
         }
         if outcomeType == "BINARY":
             data["initialProb"] = initialProb
         elif outcomeType == "FREE_RESPONSE":
             pass
-        elif outcomeType == "NUMERIC":
+        elif outcomeType == "PSEUDO_NUMERIC":
             data["min"] = minValue
             data["max"] = maxValue
+            data["isLogScale"] = isLogScale
+            if initialValue is None:
+                raise ValueError("Needs initial value")
+            data["initialValue"] = initialValue
+        elif outcomeType == "MULTIPLE_CHOICE":
+            data["answers"] = answers
         else:
             raise Exception(
-                "Invalid outcome type. Outcome should be one of: BINARY, FREE_RESPONSE, NUMERIC"
+                "Invalid outcome type. Outcome should be one of: BINARY, FREE_RESPONSE, PSEUDO_NUMERIC, MULTIPLE_CHOICE"
             )
 
         response = requests.post(
             url=BASE_URI + "/market",
             json=data,
             headers=self._auth_headers(),
         )
+        if response.status_code in range(400, 500):
+            response.raise_for_status()
+        elif response.status_code >= 500:
```

Comment:
Can you add a comment here why if there's an internal error we make a different API call?